build_tools/benchmarks/run_benchmarks_on_linux.py

```diff
@@ -109,18 +118,21 @@ def __run_benchmark(self, benchmark_case: BenchmarkCase,
               driver_info=benchmark_case.driver_info,
               benchmark_min_time=self.config.benchmark_min_time))
 
-    result_json = execute_cmd_and_get_output(
+    benchmark_stdout = execute_cmd_and_get_output(
```

Comment:
It seems to still be a good idea to write down the original google benchmark json output at some where for debugging?

Reply:
Since `BenchmarkRun.context` is still somewhat specific to Google Benchmark, maybe it would be reasonable to use that field to optionally store the raw results from each benchmark framework instead? That would also make it possible to retroactively extract additional information from the benchmarks if we later decide that there are additional metrics that we care about.

Reply:
I think it's a good idea to store the raw benchmark results in `BenchmarkRun`. But we can also dump each raw result to a file. For local debugging, it will be easier to check each individual benchmark result instead of the unified benchmark results json.

Reply:
SG! The `BenchmarkRun`s are serialized to the same location that the Google Benchmark JSONs were previously, so after updating the context as we described the local debugging information will contain all the information it used to.